SkillChat.Server/NETVersionComparison.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+
+namespace SkillChat.Server
+{
+    public class NETVersionComparison
+    {
+        /// <summary>
+        /// Получение наиболее подходящего Runtime .NET
+        /// </summary>
+        public static Runtime GetBestDotNETRuntime(string cfgVersion)
+        {
+            Runtime cfgRuntime = new Runtime() {Version = cfgVersion};
+            List<Runtime> _runtimesInstaled = new List<Runtime>();
+            List<Runtime> _NETCoreRuntimes = new List<Runtime>();
+
+            GetPackageList(_runtimesInstaled);
```

Comment:
Это плохая практика через параметр передавать список, который будешь заполнять, при этом метод ничего не возвращает. 
1. Выглядит так, будто метод ничего не должен возвращать, хотя название намекает на это
2. Тебе приходится инициализировать список снаружи вызова
3. При вызове этого метода больше одного раза, заполнится несколько раз.
4. Если тебе надо что-то вернуть, то используй возвращаемое значение. Если надо асинхронно, можно сделать асинхронный метод. Можно возвращать поэлементно если тип возвращаемого значения выбрать IEnumerable
```suggestion
            _runtimesInstaled = GetPackageList();
```